src/domain/validation-constants.js

```diff
@@ -13,7 +13,7 @@ export const minUserPasswordLength = 8;
 
 export const maxRoomShortDescriptionLength = 130;
 
-export const maxMediaLibraryItemDescriptionLength = 1000;
+export const maxMediaLibraryItemShortDescriptionLength = 160;
```

Comment:
I'd suggest to keep the 1000 as to reflect the document's short description of 1000 (as changed in https://github.com/educandu/educandu/pull/973)

Reply:
We're not consistent with this actually. Room and profile have their custom lengths (based actually on how we render them in cards). For media library items we have a max length of 158 characters in the database at the moment, from 8481 records. To me nothing here speaks for a 1000 characters limit in a "short description", I find it overkill 😬 

But in case at any point someone needs to be a little more specific, we could make it higher.

Reply:
I now realized it's about the Redaction page, where we show the full description. Ok, I can make that consistent..
Just for the record, I do dislike the abuse of a "short description", but in the end authors can be as efficient or inefficient as they want with the content.